src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
```

Comment:
The `processOperation` method in `ReturnOperation` should be independent and not directly modify the `Storage`. Instead, it should work with a `FruitDao` to interact with the storage data.